packages/fxa-auth-server/lib/routes/password.ts

```diff
@@ -555,7 +554,7 @@ module.exports = function (
 
         // TODO FXA-7852
         // await mailer.sendPasswordForgotOtpEmail(account.emails, account, {
-        const noopSendMail = () => {};
+        const noopSendMail = (...args: any) => {};
```

Comment:
nit, the ts(6113) warning about unused args can be avoided by chaning this to ..._args